lib/app-common-ruby/types.rb

```diff
@@ -1,346 +1,348 @@
 # This file is automatically generated by bin/json_schema_ruby
 require 'ostruct'
 
-class AppConfig < OpenStruct
-  attr_accessor :logging
-  attr_accessor :kafka
-  attr_accessor :database
-  attr_accessor :objectStore
-  attr_accessor :inMemoryDb
-  attr_accessor :featureFlags
-  attr_accessor :endpoints
-  attr_accessor :privateEndpoints
-
-  def initialize(attributes)
-    super
-    raise 'The input argument (attributes) must be a hash' if (!attributes || !attributes.is_a?(Hash))
-
-    attributes = attributes.each_with_object({}) do |(k, v), h|
-      raise "The input [#{k}] is invalid" unless valid_keys.include?(k.to_sym)
-      h[k.to_sym] = v
+module AppCommonRuby
```

Comment:
Same comments here: are these changes made by `RubyClassConverter`? They should be generated automatically

Reply:
@hsong-rh yes, this time I made this file by sync_config.sh